dask_saturn/core.py

```diff
@@ -66,7 +65,7 @@ class SaturnCluster(SpecCluster):
         If no size is provided, this will default to the size configured for Jupyter
         in your Saturn Cloud project.
     :param worker_is_spot: Flag to indicate if workers should be started on Spot Instances nodes.
-        Defaults to False.
+        Added in dask-saturn 0.1.0.
```

Comment:
Sorry @jameslamb - one last thing, do you think this should have a Saturn version as well?

Reply:
oh yeah, this should include the Saturn version if it's easy for users to know what Saturn version they're running